rootfs/etc/systemd/system/cec-active-source.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
Is this file needed at all? Would it be best to add WantedBy and Before/After lines in the service itself? This seems like a dirty hack.